dev-scripts/openshift_deploy.sh

```diff
@@ -0,0 +1,234 @@
+[0m [0m [0m [0m [38;5;23m0[38;5;60m0[38;5;60m0[38;5;67m0[38;5;103m0[38;5;103m0[38;5;67m0[38;5;67m1[38;5;67m1[38;5;67m1[38;5;67m1[38;5;67m0[38;5;67m0[38;5;67m1[38;5;67m0[38;5;67m0[38;5;67m0[38;5;67m0[38;5;67m0[38;5;67m1[38;5;67m1[38;5;67m1[38;5;67m0[38;5;67m1[38;5;67m1[38;5;67m1[38;5;103m1[38;5;103m0[38;5;67m0[38;5;60m0[38;5;60m0[38;5;23m0[0m [0m [0m [0m 
+[0m [0m [0m [0m [0m [0m [0m [0m [38;5;17m0[38;5;59m0[38;5;60m0[38;5;67m1[38;5;103m1[38;5;103m0[38;5;103m1[38;5;67m0[38;5;67m1[38;5;67m1[38;5;67m0[38;5;67m1[38;5;67m0[38;5;67m0[38;5;67m0[38;5;67m1[38;5;67m0[38;5;103m0[38;5;103m0[38;5;103m1[38;5;67m1[38;5;60m1[38;5;60m1[38;5;17m0[0m [0m [0m [0m [0m [0m [0m [0m 
+[0m [0m [0m [0m [0m [0m [0m [0m [0m [0m [0m [0m [38;5;17m1[38;5;59m1[38;5;60m0[38;5;60m1[38;5;67m0[38;5;103m1[38;5;103m1[38;5;67m1[38;5;67m0[38;5;103m0[38;5;103m0[38;5;67m0[38;5;60m1[38;5;60m0[38;5;59m0[38;5;17m1[0m [0m [0m [0m [0m [0m [0m [0m [0m [0m [0m [0m 
+[0m [0m [0m [0m [0m [0m [0m [0m [0m [0m [0m [0m [0m [0m [0m [0m [0m [38;5;17m0[38;5;60m0[38;5;60m0[38;5;60m1[38;5;60m0[38;5;17m1[0m [0m [0m [0m [0m [0m [0m [0m [0m [0m [0m [0m [0m [0m [0m [0m [0m 
+[0m
+EOF
+echo
+
+# --------------------------------------------------------
+# Set configuration common to both minishift and openshift
+# --------------------------------------------------------
+
+DEFAULT_CHE_IMAGE_REPO="docker.io/rhchestage/che-server"
+CHE_IMAGE_REPO=${CHE_IMAGE_REPO:-${DEFAULT_CHE_IMAGE_REPO}}
+DEFAULT_CHE_IMAGE_TAG="nightly-fabric8"
+CHE_IMAGE_TAG=${CHE_IMAGE_TAG:-${DEFAULT_CHE_IMAGE_TAG}}
+DEFAULT_CHE_LOG_LEVEL="DEBUG"
+CHE_LOG_LEVEL=${CHE_LOG_LEVEL:-${DEFAULT_CHE_LOG_LEVEL}}
+DEFAULT_CHE_DEBUGGING_ENABLED="true"
+CHE_DEBUGGING_ENABLED==${CHE_DEBUGGING_ENABLED:-${DEFAULT_CHE_DEBUGGING_ENABLED}}
+
+# OPENSHIFT_FLAVOR can be minishift or openshift
+# TODO Set flavour via a parameter
+DEFAULT_OPENSHIFT_FLAVOR=minishift
+OPENSHIFT_FLAVOR=${OPENSHIFT_FLAVOR:-${DEFAULT_OPENSHIFT_FLAVOR}}
+
+
+if [ "${OPENSHIFT_FLAVOR}" == "minishift" ]; then
+  # ---------------------------
+  # Set minishift configuration
+  # ---------------------------
+  echo -n "[CHE] Checking if minishift is running..."
+  minishift status | grep -q "Running" ||(echo "Minishift is not running. Aborting"; exit 1)
+  echo "done!"  
+
+  DEFAULT_OPENSHIFT_ENDPOINT="https://$(minishift ip):8443/"
+  OPENSHIFT_ENDPOINT=${OPENSHIFT_ENDPOINT:-${DEFAULT_OPENSHIFT_ENDPOINT}}
+  DEFAULT_OPENSHIFT_USERNAME="developer"
+  OPENSHIFT_USERNAME=${OPENSHIFT_USERNAME:-${DEFAULT_OPENSHIFT_USERNAME}}
+  DEFAULT_OPENSHIFT_PASSWORD="developer"
+  OPENSHIFT_PASSWORD=${OPENSHIFT_PASSWORD:-${DEFAULT_OPENSHIFT_PASSWORD}}
+  DEFAULT_OPENSHIFT_NAMESPACE_URL="${CHE_OPENSHIFT_PROJECT}.$(minishift ip).nip.io"
```

Comment:
shouldn't this line be *after* line 88 (where `CHE_OPENSHIFT_PROJECT` is defined) ?